src/openzeppelin/tests/account/test_account.cairo

```diff
@@ -438,8 +448,19 @@ fn test_public_key_setter_and_getter() {
     testing::set_contract_address(ACCOUNT_ADDRESS());
     testing::set_caller_address(ACCOUNT_ADDRESS());
 
+    // Check default
+    let public_key = Account::PublicKeyImpl::get_public_key(@state);
```

Comment:
```suggestion
    let public_key = PublicKeyImpl::get_public_key(@state);
```

Remove Account prefix (with imports) for consistency. Grouping the imports (merging) makes sense to me. This last part is probably better for another PR for getting consensus on the pattern.

Reply:
Applies for the rest of the file as well.